lib/ex_aws/sqs.ex

```diff
@@ -307,6 +312,13 @@ defmodule ExAws.SQS do
     |> ExAws.Utils.camelize
   end
 
+  defp format_message_attributes(attributes) do
+    attributes
+    |> Enum.with_index
+    |> Enum.map(&format_message_attribute/1)
+    |> Enum.reduce(%{}, &Map.merge(&1, &2))
```

Comment:
This really should just be a Map.new or Enum.into instead of a map with a reduce via merge. IE
```
attributes
|> Enum.with_index(1)
|> Map.new(fn {attr, i} ->
  {"MessageAttributeNames.#{i}", to_string(attr)}
end)
```